prisma/schema.prisma

```diff
@@ -152,3 +153,14 @@ model Comment {
 
   @@map("Comments")
 }
+
+model LabelMapping {
+  id             String    @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  label          String    @unique
+  labelledEntity String
+  createdAt      DateTime  @default(now()) @db.Timestamptz()
+  updatedAt      DateTime  @updatedAt @db.Timestamptz()
+  deletedAt      DateTime? @db.Timestamptz()
+
+  @@map("LabelMappings")
+}
```

Comment:
Just a suggestion, can we rename this table to something more intuitive like `Labels`? That is if the name LabelMappings isn't specifically recommended by the Copilot team in their requirements

Reply:
Changed in the commits below